Mage.Sets/src/mage/cards/c/CallousOppressor.java

```diff
@@ -0,0 +1,160 @@
+import mage.MageInt;
+import mage.MageObject;
+import mage.abilities.Ability;
+import mage.abilities.common.AsEntersBattlefieldAbility;
+import mage.abilities.common.SimpleActivatedAbility;
+import mage.abilities.common.SkipUntapOptionalAbility;
+import mage.abilities.condition.common.SourceTappedCondition;
+import mage.abilities.costs.common.TapSourceCost;
+import mage.abilities.decorator.ConditionalContinuousEffect;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.continuous.GainControlTargetEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.choices.Choice;
+import mage.choices.ChoiceImpl;
+import mage.constants.CardType;
+import mage.constants.SubType;
+import mage.constants.Duration;
+import mage.constants.Outcome;
+import mage.constants.Zone;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.predicate.Predicates;
+import mage.filter.predicate.mageobject.ChosenSubtypePredicate;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.common.TargetCreaturePermanent;
+import mage.target.common.TargetOpponent;
+import mage.util.CardUtil;
+
+/**
+ *
+ * @author L_J
+ */
+public class CallousOppressor extends CardImpl {
+
+    public CallousOppressor(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId,setInfo,new CardType[]{CardType.CREATURE},"{1}{U}{U}");
+        this.subtype.add(SubType.CEPHALID);
+        this.power = new MageInt(1);
+        this.toughness = new MageInt(2);
+
+        // You may choose not to untap Callous Oppressor during your untap step.
+        this.addAbility(new SkipUntapOptionalAbility());
+
+        // As Callous Oppressor enters the battlefield, an opponent chooses a creature type.
+        this.addAbility(new AsEntersBattlefieldAbility(new OpponentChooseCreatureTypeEffect()));
+
+        // {T}: Gain control of target creature that isn't of the chosen type for as long as Callous Oppressor remains tapped.
+        FilterCreaturePermanent filter = new FilterCreaturePermanent("creature that isn't of the chosen type");
+        filter.add(Predicates.not(new ChosenSubtypePredicate(this.getId())));
+        ConditionalContinuousEffect effect = new ConditionalContinuousEffect(
+                new GainControlTargetEffect(Duration.OneUse),
+                SourceTappedCondition.instance,
+                "Gain control of target creature for as long as Callous Oppressor remains tapped");
+        Ability ability = new SimpleActivatedAbility(Zone.BATTLEFIELD, effect, new TapSourceCost());
+        ability.addTarget(new TargetCreaturePermanent(filter));
+        this.addAbility(ability);
+    }
+
+    public CallousOppressor(final CallousOppressor card) {
+        super(card);
+    }
+
+    @Override
+    public CallousOppressor copy() {
+        return new CallousOppressor(this);
+    }
+}
+
+class OpponentChooseCreatureTypeEffect extends OneShotEffect {
```

Comment:
With this name this should be a class of the main repository. 
Otherwise call it CallousOppressorChooseTypeEffect if you locate it here.